### PR TITLE
cairo: remove non-installed files

### PR DESCRIPTION
**Pull request by tobsan**
Part of preparation for release of a pyro branch

Signed-off-by: Tobias Olausson <tobias.olausson@pelagicore.com>

### CODE REVIEW

File: recipes-temporary-patches/cairo/cairo_%.bbappend [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
+# QA issues with installed but not shipped files
+# This is fixed in meta-ivi but not in 13.0 branch (pyro)
+
+do_install_append() {
+   rm ${D}/usr/lib/cairo/libcairo-trace.la
```
Comment (GordanM) [inline recipes-temporary-patches/cairo/cairo_%.bbappend]: Maybe using `{libdir}` would be better than `/usr/lib/.

Comment (DunderRoffe) [reply recipes-temporary-patches/cairo/cairo_%.bbappend]: +1

**Comment by tobsan** [comment]
@GordanM This is fixed in master on meta-ivi, but not in 13.0 branch, which meta-bistro and meta-pelux syncs against (or that's the plan at least)

**Comment by GordanM** [comment]
@tobsan what is the reason to not sync against master? meta-ivi always had a mind of it's own not really following Yocto/OE releases. Syncing against master at this point would be beneficial because some Pelagicore contributions were made to ensure the `pyro` build AFAIK.

**Comment by tobsan** [comment]
@GordanM because the meta-ivi releases still follows yocto releases somehow, and to be able to branch off master to pyro from meta-bistro and meta-pelux, I need them to be in sync with the yocto release branches for the layers they depend on. Once that is done, master should prepare for the next yocto release imho (and later branch off to that one). Following master in one place but not all makes little sense. It is bad enough that meta-ivi and meta-qt5 has different branching names/strategies already :)

**Comment by GordanM** [comment]
@tobsan fair enough. Though make sure to poke the `meta-ivi` maintainer to cherry-pick the patch to 13.0 and then I am fine with merging this **temporary** patch.

**Pull request merged**